food-and-agriculture-organization-of-the-united-nations.csl

```diff
@@ -0,0 +1,217 @@
+    </sort>
+    <layout>
+      <text macro="author" font-weight="bold"/>
+      <choose>
+        <if variable="issued">
+          <date variable="issued" suffix=". ">
+            <date-part name="year"/>
+          </date>
+        </if>
+        <else>
+          <text value="undated" text-case="capitalize-first" suffix=". "/>
+        </else>
+      </choose>
+      <choose>
+        <if type="bill book graphic legal_case legislation motion_picture song" match="any">
+          <choose>
+            <if match="any" is-numeric="edition">
+              <group delimiter=", " prefix=" " suffix=". ">
+                <text variable="title" font-style="italic"/>
+                <text macro="edition"/>
+              </group>
+              <text macro="editor"/>
+              <text macro="publisher"/>
+            </if>
+            <else>
+              <text variable="title" font-style="italic" suffix="."/>
+              <text macro="editor"/>
+              <text macro="publisher"/>
+            </else>
+          </choose>
+        </if>
+        <else-if type="chapter" match="any">
+          <text variable="title" prefix=" " suffix=". "/>
+          <text term="in" text-case="capitalize-first" font-style="italic"/>
+          <group delimiter=" " prefix=" ">
+            <text macro="editor"/>
+            <text variable="container-title" font-style="italic" suffix=", "/>
+            <text variable="collection-title" prefix=" " suffix="."/>
+          </group>
+          <group suffix=".">
+            <label variable="page" suffix="." form="short"/>
+            <text variable="page"/>
+          </group>
+          <group suffix=".">
+            <text macro="publisher" prefix=" "/>
+            <group prefix=", ">
+              <text variable="number-of-pages" suffix=" "/>
+              <label variable="page" form="short"/>
+            </group>
+          </group>
+        </else-if>
+        <else-if type="webpage" match="any">
+          <text variable="title" prefix=" " suffix=". "/>
+          <text macro="Available-at-and-accessed"/>
+        </else-if>
+        <else-if type="article-newspaper article-magazine" match="any">
+          <text variable="title" suffix=". "/>
+          <text variable="container-title" form="short" font-style="italic" suffix="."/>
+          <group prefix=" ">
+            <choose>
+              <if match="any" variable="URL">
+                <text macro="Available-at-and-accessed" prefix=" "/>
+              </if>
+              <else>
+			    <text value=""/>
+			  </else>
+            </choose>
+          </group>
+        </else-if>
+        <else-if type="report" match="any">
+          <text variable="title" font-style="italic" suffix="."/>
+          <text macro="publisher"/>
+          <group prefix=" ">
+            <choose>
+              <if match="any" variable="URL">
+                <text macro="Available-at-and-accessed"/>
+              </if>
+              <else>
```

Comment:
same here for 177-179: an "else don't print anything" is always implied in if/else-if elements.